src/System.Private.CoreLib/shared/System/Reflection/Assembly.cs

```diff
@@ -37,6 +37,10 @@ public virtual IEnumerable<TypeInfo> DefinedTypes
         public virtual Type[] GetTypes()
         {
             Module[] m = GetModules(false);
```

Comment:
Care to name the parameter for this `false`? Below as well? Please?

Reply:
I agree named arguments are helpful, but since that's about style and I wasn't changing this code anyway, I've left it as is.

Reply:
😭